pkg/cmd/adm/register_member.go

```diff
@@ -509,6 +518,20 @@ until the SpaceProvisionerConfig.spec.enabled is set to true.
 `, v.hostClusterData.apiEndpoint))
 }
 
+func (v *registerMemberValidated) getRegMemConfigFlagsAndClient(ctx *clicontext.CommandContext, clusterName string) (kubeConfigFlag *genericclioptions.ConfigFlags, rccl runtimeclient.Client, err error) {
```

Comment:
minor - do you think it makes sense to cover this with a test?

Reply:
I thought about it, but couldn't find a easier way to test it, and also the similar functionality we did not test it in restart.go.